From 89f71bd6decf104d14bddef0a1b84b64a7991d7e Mon Sep 17 00:00:00 2001
From: Haoran Wang <b50027@freescale.com>
Date: Fri, 12 Jun 2015 18:33:58 +0800
Subject: [PATCH 1/2] MA-6791 Modify ethernet score to increase priority

As the wifi AP may have a 100 score when the device
boot up without ethernet connection and it may take over
the Ethernet. But once the Ethernet enabled before Wifi
the Ethernet will be OK.

So we modify the ethernet's score to 150 that will always
be larger than 100 even it is unvalidated which will be
minus 40 (110).

Signed-off-by: Haoran Wang <b50027@freescale.com>
---
 java/com/android/server/ethernet/EthernetNetworkFactory.java | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/java/com/android/server/ethernet/EthernetNetworkFactory.java b/java/com/android/server/ethernet/EthernetNetworkFactory.java
index 6074dda..495cf7a 100644
--- a/java/com/android/server/ethernet/EthernetNetworkFactory.java
+++ b/java/com/android/server/ethernet/EthernetNetworkFactory.java
@@ -1,5 +1,6 @@
 /*
  * Copyright (C) 2014 The Android Open Source Project
+ * Copyright (C) 2015 Freescale Semiconductor, Inc.
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
@@ -70,7 +71,7 @@ import java.io.PrintWriter;
 class EthernetNetworkFactory {
     private static final String NETWORK_TYPE = "Ethernet";
     private static final String TAG = "EthernetNetworkFactory";
-    private static final int NETWORK_SCORE = 70;
+    private static final int NETWORK_SCORE = 150;
     private static final boolean DBG = true;
 
     /** Tracks interface changes. Called from NetworkManagementService. */
-- 
1.8.0

